Detectors/ITSMFT/ITS/base/src/GeometryTGeo.cxx

```diff
@@ -853,19 +856,21 @@ int GeometryTGeo::extractLayerChipType(int lay) const
 //__________________________________________________________________________
 void GeometryTGeo::Print(Option_t*) const
 {
-  printf("NLayers:%d NChips:%d\n", mNumberOfLayers, getNumberOfChips());
   if (!isBuilt()) {
+    LOGF(info, "Geometry not built not yet!");
     return;
   }
 
+  LOGF(info, "Summary of GeometryTGeo: %s", getName());
+  LOGF(info, "NLayers:%d NChips:%d\n", mNumberOfLayers, getNumberOfChips());
   for (int i = 0; i < mNumberOfLayers; i++) {
-    printf(
-      "Lr%2d\tNStav:%2d\tNChips:%2d "
-      "(%dx%-2d)\tNMod:%d\tNSubSt:%d\tNSt:%3d\tChip#:%5d:%-5d\tWrapVol:%d\n",
-      i, mNumberOfStaves[i], mNumberOfChipsPerModule[i], mNumberOfChipRowsPerModule[i],
-      mNumberOfChipRowsPerModule[i] ? mNumberOfChipsPerModule[i] / mNumberOfChipRowsPerModule[i] : 0,
-      mNumberOfModules[i], mNumberOfHalfStaves[i], mNumberOfStaves[i], getFirstChipIndex(i), getLastChipIndex(i),
-      mLayerToWrapper[i]);
+    LOGF(info,
```

Comment:
Do we really need this to be logged?

Reply:
Probably not, if you prefer it. The only use case I see of this `Print()` is anyways in the IT3 digitizer, that's why I wanted to log this.